src/main/webapp/js/utils/form/form-utils.jsx

```diff
@@ -115,15 +115,23 @@ const renderFormField = (props) => {
   );
 };
 
-export const validate = fieldConfig => (values) => {
+export const validate = (fieldConfig, entities, entityToEdit) => (values) => {
   const errors = {};
-
   _.forEach(fieldConfig, (config) => {
     const val = values[config.name];
 
     if (config.required && (_.isNil(val) || val === '')) {
       errors[config.name] = `${config.displayName} is required`;
     }
+    if (config.uniqueField) {
+      const prevValue = entityToEdit ? entityToEdit[config.name] : null;
+      const isUniqueField = _.find(
+        entities, entity => (entity[config.name] === val && prevValue !== val),
```

Comment:
you don't need the prevValue here you can check if the entity.id and values.id are different instead to be sure it's not the same entity